source/isaaclab/isaaclab/envs/manager_based_rl_env.py

```diff
@@ -102,11 +105,20 @@ def max_episode_length(self) -> int:
         """Maximum episode length in environment steps."""
         return math.ceil(self.max_episode_length_s / self.step_dt)
 
+    @property
+    def curr_episode_length(self) -> torch.Tensor:
```

Comment:
`episode_length_buf` is a variable you can access outside as it is not a private member. I don't think making a property for it is necessary at this point.